Source/LinqToDB/Linq/Builder/ExpressionTestGenerator.cs

```diff
@@ -534,7 +532,7 @@ void BuildType(Type type)
 			var ctors = type.GetConstructors().Select(c =>
 			{
 				var attrs = c.GetCustomAttributesData();
-				var attr  = attrs.Count > 0 ? attrs.Select(a => "\r\n\t\t" + a.ToString()).Aggregate((a1,a2) => a1 + a2) : "";
+				var attr  = attrs.Count > 0 ? string.Join(string.Empty, attrs.Select(a => "\r\n\t\t" + a.ToString())) : string.Empty;
```

Comment:
Technically, if `attrs.Count == 0`, the result of `string.Join()` is `string.Empty`.
```suggestion
				var attr  = string.Join(string.Empty, attrs.Select(a => "\r\n\t\t" + a.ToString()));
```

Reply:
Repeated throughout file.